my-express-app/server.js

```diff
@@ -51,7 +57,53 @@ app.get("/videos", async (req, res) => {
     res.status(500).json({ message: err.message });
   }
 });
+const storage = multer.diskStorage({
+  destination: function (req, file, cb) {
+    cb(null, "uploads/");
+  },
+  filename: function (req, file, cb) {
+    cb(null, Date.now() + path.extname(file.originalname));
+  },
+});
 
+const upload = multer({
```

Comment:
Be prepared to explain these lines of codes in our next technical challenge discussion! Curious to here what these lines of code do

Reply:
Okay !